app/models/task.rb

```diff
@@ -13,4 +13,9 @@ def service_options
 
   def dispatch
   end
+
+  def timed_out?
```

Comment:
@hsong-rh We should test this using this gem
https://github.com/travisjeffery/timecop

Which will move the time without us having to wait 10 minutes